media/static-bundles.json

```diff
@@ -1311,6 +1317,13 @@
       ],
       "name": "firefox_new_scene1_berlin"
     },
+    {
+      "files": [
+        "js/firefox/new/variation-scene1.js",
+        "js/firefox/new/de-experiment.js"
```

Comment:
I'm not in love with the idea of having this file included in `de/scene1-b.html`, as most of the code will never pertain to/execute on that page. It's a _little_ messy. I don't see a way it can fail though, so up to you. (It is just a temporary experiment.) 😄

Reply:
I think this just gets to live like this until the test is over :/